azure_functions_worker/dispatcher.py

```
@@ -309,13 +314,24 @@ async def _handle__worker_init_request(self, request):
             except Exception as ex:
                 self._function_metadata_exception = ex
 
+            if sys.version_info.minor >= BASE_EXT_SUPPORTED_PY_MINOR_VERSION \
+                    and self._has_http_func:
+                from azure.functions.extension.base \
```

fail fast try catch return error streaming